### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by seegwen**
This will remove useless code and fix some indentations and trailing comments

### CODE REVIEW

File: src/views/Login.vue [hunk 1]
```diff
@@ -83,9 +83,7 @@ import { RestExc } from '@lokavaluto/lokapi-browser';
           login: this.email,
           password: this.password,
         });
-        this.$store.dispatch("setAccounts");
         this.success = "Connection réussie";
-        this.$store.state.lokapi.isLog = true;
```
Comment (vaab) [inline src/views/Login.vue:88]: I'm under the impression that all usages of `isLog` are useless. Can we have one commit removing all of them in all the code in one go ?

**Review by vaab — CHANGES_REQUESTED** [review]

**Comment by vaab** [comment]
As I have commented only one commit and that all the other make sense for me, I'll push all the others if you don't mind meanwhile.

**Comment by vaab** [comment]
I've reviewed, tested and pushed to main all your commit except aa05ed3f88f71966db467d2a1bf210623f8353b0 .

File: src/views/Login.vue [hunk 2]
```diff
@@ -83,9 +83,7 @@ import { RestExc } from '@lokavaluto/lokapi-browser';
           login: this.email,
           password: this.password,
         });
-        this.$store.dispatch("setAccounts");
```
[the remark this answers is not in the document]
Comment (seegwen) [reply src/views/Login.vue:86]: Ok, do not. I'm removing this commit.

File: src/views/Login.vue (hunk 1 above)
Comment (seegwen) [reply src/views/Login.vue:88]: If you want.

Comment (seegwen) [reply src/views/Login.vue:88]: So I only kept this change in a new commit at the end of the branch.
I discarded the other change.

**Comment by vaab** [comment]
Thanks a lot, the last commit that remained to push is integrated and ready for push.

**Pull request closed without merging**